dev-tools/packer/platforms/centos/init.j2

```diff
@@ -54,6 +58,9 @@ start() {
     daemon $daemonopts $wrapper $wrapperopts -- $agent $args
     RETVAL=$?
     echo
+    if [ $oom_adj -lt 1000 ]; then
+        echo "$oom_adj" > "/proc/$(pidof packetbeat)/oom_score_adj"
```

Comment:
Instead of using `pidof packetbeat` can we get the PID from the pidfile? It seems like that would be more robust if more than one instance of packetbeat are running.